### DISCUSSION TITLE
cant use sweetkind theme

### DISCUSSION THREAD

**Discussion by plugeit** [TomSchimansky/CustomTkinter#920] [Q&A]
FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/RepulsiveTroubledAssociate/venv/lib/python3.8/site-packages/customtkinter/windows/widgets/theme/../../../assets/themes/sweetkind.json'

**Comment by Akascape** [comment]
@icanhackyou **Sweetkind** theme is removed. May add it in the next update.
